packages/@react-spectrum/cards/src/GalleryLayout.tsx

```diff
@@ -117,16 +174,35 @@ export class GalleryLayout<T> extends BaseLayout<T> {
     // Compute aspect ratios for all of the items, and the total width if all items were on in a single row.
     let ratios = [];
     let totalWidth = 0;
+    let minRatio = this.minItemSize.width / this.minItemSize.height;
```

Comment:
Note that this still has a bit of an issue where it is possible for the item's height for extremely tall images to be a bit taller than its row peers:
![image](https://user-images.githubusercontent.com/9661127/133529439-60e23cac-3953-45e9-957f-a0f8d8873bbd.png)

Reply:
well that doesn't seem good, lets sit down and go through the math together sometime